### PR TITLE
Remove duplicates from size conditions

### PR DESCRIPTION
**Pull request by babich-a**

### CODE REVIEW

File: scss/widgets/generic/gantt/_sizes.scss [hunk 1]
```diff
@@ -17,7 +17,6 @@ $gantt-task-wrapper-padding-top: 0;
 @if $size == "default" {
   $gantt-task-title-padding-left: 7px;
   $gantt-task-height: 21px;
-  $gantt-parent-task-height: 19px;
```
Comment (San4es) [inline scss/widgets/generic/gantt/_sizes.scss:20]: Why should we remove this?

Comment (babich-a) [reply scss/widgets/generic/gantt/_sizes.scss:20]: Unused variable

Comment (San4es) [reply scss/widgets/generic/gantt/_sizes.scss:20]: Ok. So, should we remove this line as well? https://github.com/DevExpress/DevExtreme/pull/13932/files/92cb5f0fd6068f1a98025bfea05708e486bf67ae#diff-da9a39ef2955420a3c08024f14faa5deL7

Comment (babich-a) [reply scss/widgets/generic/gantt/_sizes.scss:20]: removed

**Review by San4es — APPROVED** [review]

**Pull request merged**